Client/Clique/Modules/CreatedEvents/CreatedEventsViewController.swift

```diff
@@ -8,7 +8,72 @@
 import UIKit
 
 class CreatedEventsViewController: UIViewController {
+    
+    @IBOutlet private var _tableView : UITableView?
+    
+    private var _events : [Event] = [] {
+        didSet {
+            _tableView?.reloadData()
+        }
+    }
+    private var _eventServices = EventServices()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupUI()
+        _getEvents()
+    }
+}
+
+private extension CreatedEventsViewController {
+    func setupUI(){
+        _tableView?.delegate = self
+        _tableView?.dataSource = self
+        _tableView!.register(UINib(nibName: Constants.Storyboards.eventListTableViewCell, bundle: nil), forCellReuseIdentifier: Constants.Storyboards.eventListTableViewCell)
+    }
+    
+    func _getEvents(){
+        _eventServices.getLogUserEvents() { [weak self] result in
+            guard let _viewController = self else { return }
+            switch result{
+            case .success(let events):
+                _viewController._events = events
+            case .failure(let error):
+                print(error)
+                
```

Comment:
Currently, error handling is not defined. If events are not fetched correctly, you should show something to the user - e.g., an error message with the try again button.

This way, you will print the error in the console, which should not be in the production code. Please revisit this and add the fallback implementation when this API call fails.

Reply:
This is not fixed, I'm just adding a comment here to not forget about it.